rust/relay/ebpf-turn-router/src/main.rs

```diff
@@ -84,37 +85,46 @@ pub fn handle_turn(ctx: XdpContext) -> u32 {
     try_handle_turn(&ctx).unwrap_or_else(|e| match e {
         Error::NotIp | Error::NotUdp => xdp_action::XDP_PASS,
 
-        Error::PacketTooShort
+        Error::PacketTooLong
+        | Error::PacketTooShort
         | Error::NotTurn
         | Error::NotAChannelDataMessage
         | Error::Ipv4PacketWithOptions
         | Error::NoMacAddress
-        | Error::UnsupportedChannel(_)
         | Error::NoEntry(_) => {
             debug!(&ctx, "Passing packet to userspace: {}", e);
-
             xdp_action::XDP_PASS
         }
 
         Error::BadChannelDataLength
         | Error::XdpStoreBytesFailed(_)
-        | Error::XdpAdjustHeadFailed(_)
+        | Error::XdpAdjustTailFailed(_)
+        | Error::UnsupportedChannel(_)
```

Comment:
I did confirm in fact that moving this one to `XDP_DROP` completely blocks all connectivity for my ipv4 client/gateway.